nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,115 @@
+#ICA_AROMA pulled from: https://github.com/rhr-pruim/ICA-AROMA
+#This assumes ICA_AROMA.py is already installed and callable via $PATH
+from nipype.interfaces.base import (
+    TraitedSpec,
+    CommandLineInputSpec,
+    CommandLine,
+    File,
+    Directory,
+    traits,
+    OutputMultiPath
+)
+import os
+
+class ICA_AROMAInputSpec(CommandLineInputSpec):
+    featDir = Directory(exists=True,
```

Comment:
If it's non-mandatory, don't include it. But since you're in the situation where you have to either have `feat_dir` or `in_file`, then both should be mandatory, but `xor` one another. (The mandatory check is smart enough to figure out these required-but-mutually-exclusive cases.)

I think your motion correction params, affine transform and warp are all required (if not using `feat_dir`), so they should be set to mandatory as well, but the mask is generated by the program if not supplied, so it doesn't need to be set to mandatory.